src/checked_runner.ml

```diff
@@ -318,6 +318,16 @@ module Make (Backend : Backend_extended.S) = struct
         let s, y = next_auxiliary s in
         run (k y) s
 
+  let run t s =
+    try run t s
+    with exn ->
+      Printf.eprintf
```

Comment:
Hmm I am a bit skeptical about just writing to stderr. Maybe this should return a more structured error object and the caller can print?